### DISCUSSION TITLE
A nice new technology

### DISCUSSION THREAD

**Discussion by zhaoqi571436204** [comfyanonymous/ComfyUI#5547] [Ideas]
https://github.com/mit-han-lab/nunchaku
https://huggingface.co/mit-han-lab/svdquant-models
VDQuant is a post-training quantization technique for 4-bit weights and activations that well maintains visual fidelity. On 12B FLUX.1-dev, it achieves 3.6× memory reduction compared to the BF16 model. By eliminating CPU offloading, it offers 8.7× speedup over the 16-bit model when on a 16GB laptop 4090 GPU, 3× faster than the NF4 W4A16 baseline. O

**Comment by Woukim** [comment]
@comfyanonymous please will this be implemented in the ComfyUI?🙏
